libs9s/s9srpcclient.cpp

```diff
@@ -3134,8 +3134,7 @@ S9sRpcClient::createCluster()
     osUserName     = options->osUser();
     vendor         = options->vendor();
 
-
```

Comment:
what about the postgresql culster type use case. it is not needed here?

Reply:
Now **--vendor** value should be provided for postgres cluster create command, like we have for other clusters, e.g. for mongo cluster type.